website/docs/plugin/testing/acceptance-tests/configuration.mdx

```diff
@@ -0,0 +1,277 @@
+---
```

Comment:
Do we want to callout this documentation page on the existing `/terraform/plugin/testing/acceptance-tests/teststep#test-modes` page when talking about `Lifecycle` mode? 

All the examples in that file use the raw `Config`, but that's probably fine